mnemonic.c

```diff
@@ -450,10 +450,12 @@ mn_decode (char *src, void *dest, int destsize)
 
   while ((index = mn_next_word_index (&src)) != 0)
     {
-      if (index == 0 && *src != 0)
-	return MN_EWORD;
       (void) mn_decode_word_index (index, dest, destsize, &offset);
```

Comment:
The docs say you only have to check the error code of the _last_ call to `mn_decode_word_index`.